rgw/v2/tests/nfs_ganesha/config/rgw_user.yaml

```diff
@@ -1,9 +1,12 @@
-user_id:
-access_key:
-secret_key:
-rgw_hostname:
-ganesha_config_exists: true # set true to create ganesha config based on the above input or set false not to create any config.
-already_mounted: # set true if nfs_ganesha is already mounted or false if not.
-nfs_version:
-nfs_mnt_point: # provide full path
-Pseudo:
+---
+user_id:  # Provide the User_Id from /etc/ganesha/ganesha.conf
+access_key:  # Provide the Access_Key_Id value from /etc/ganesha/ganesha.conf for the above User_Id
+secret_key:  # Provide the Secret_Access_Key value from /etc/ganesha/ganesha.conf for the above User_Id
+rgw_hostname:  # Provide the hostname of the rgw node
+ganesha_config_exists: true  # set true to create ganesha config based on the above input or set false not to create any config.
+already_mounted:  # set true if nfs_ganesha is already mounted or false if not.
+cleanup:  # By default set to true to perform a cleanup after a test is executed successfully.
+do_unmount:  # By default set to true to perform an unmount on the ganesha mount_point after a test is executed successfully.
+nfs_version:  # Provide the nfs protocol version being used
```

Comment:
mention in the comment, to user `4` or `3` and not `4.1` or `3.1` etc.

Reply:
Added the changes, mentioning Accepted values : 3 or 4